### PR TITLE
Registration Spacing Fix Mobile

### PR DESCRIPTION
**Pull request by martypowell**
Ensure there is space between the registration button registration required message on mobile devices.

### CODE REVIEW

**Comment by ascire** [comment]
What's up with all the failing checks?

File: dist/js/master.js [hunk 1]
```diff
@@ -1408,12 +1414,7 @@ bcpl.navigation = function ($, keyCodes) {
...
-	$(document).on('keydown', '#responsive-sliding-navigation button', navigationButtonKeyPressed);
-	$(document).on('keydown', '#responsive-sliding-navigation', navigationKeyPressed);
-	$(document).on('click', navButtonSelector, navButtonClicked);
-	$(document).on('keydown', '#responsive-sliding-navigation a', navigationMenuItemKeyPressed);
+	$(document).on('mouseover', '.nav-and-search:not(.search-is-active) #responsive-sliding-navigation button, #responsive-sliding-navigation .submenu-wrapper', navigationMouseover).on('mouseleave', '.nav-and-search:not(.search-is-active) #responsive-sliding-navigation button, #responsive-sliding-navigation .submenu-wrapper', navigationMouseleave).on('keydown', '#responsive-sliding-navigation button', navigationButtonKeyPressed).on('keydown', '#responsive-sliding-navigation', navigationKeyPressed).on('click', navButtonSelector, navButtonClicked).on('click', backButtonSelector, onBackButtonClicked).on('keydown', '#responsive-sliding-navigation a', navigationMenuItemKeyPressed);
```
Comment (ascire) [inline dist/js/master.js]: I feel like I've already reviewed this change.

But in case I haven't, add some whitespace for God's sake.

Comment (martypowell) [reply dist/js/master.js]: hey @ascire take a gander at the file location on your comment 😆

Comment (ascire) [reply dist/js/master.js]: ...Welp.

**Pull request merged**